modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/CurrencySnapshotAcceptanceManager.scala

```diff
@@ -249,7 +249,7 @@ object CurrencySnapshotAcceptanceManager {
       )
 
       acceptedTokenLocks = incomingTokenLocks
-        .filter(itl => itl.unlockEpoch >= epochProgress)
+        .filter(itl => itl.unlockEpoch.forall(_ >= epochProgress))
```

Comment:
`forall` returns `true` for `None`. is that correct?

Reply:
I think this makes sense, unlock is being validated only if there's an unlock provided, otherwise it has no criteria.